### PR TITLE
Skip cudnn test when cudnn v2 is installed

### PR DESCRIPTION
**Pull request by unnonouno**
cuDNN is used only when cuDNN ver >=3 is installed on softmax cross entropy.

### CODE REVIEW

**Comment by okuta** [comment]
LGTM!
